Makefile

```diff
@@ -209,6 +213,39 @@ test-e2e: KUSTOMIZE_BUILD_DIR := config/overlays/e2e
 test-e2e: GO_BUILD_FLAGS := -cover
 test-e2e: run image-registry e2e e2e-coverage kind-clean #HELP Run e2e test suite on local kind cluster
```

Comment:
If I'm reading the Makefile correctly, we have:
`test-e2e` - runs `operator-controller` e2e only
`catalogd-e2e` - runs `catalogd` e2e only

I think we need some more clear separation between the e2e for the two components. I'd prefer this as a follow-up if we do change it, but IMO we should have:
`test-e2e` - runs **both** `operator-controller` and `catalogd` e2e
`catalogd-e2e` - runs`catalogd` e2e only
`operator-controller-e2e` - runs `operator-controller` e2e only

We could then have the github actions run the `catalogd-e2e` and `operator-controller-e2e` in two separate jobs as it does now, but with slightly more clear job names. WDYT?

Again though, don't do that on this PR if you agree, I think this one is big enough already :)

Reply:
HI @dtfranz 

I also want :-) That is what I expected we have at some point:

> test-e2e - runs both operator-controller and catalogd e2e
> upgrade-e2e runs for both

But that needs to be follow ups.
We need to do step by step

- We need change the tests to no longer use Omega Ginkgo
- Then, we can start to began to combine
- We will need to combine as well the image registry and going on

We need to split. Otherwise, it is too much for a single go